src/renderers/dom.js

```diff
@@ -162,6 +166,24 @@ const getDefaultDomOptions = () : DomOptions => {
     return {};
 };
 
+function addXmlNamespace(node : ElementNode | TextNode | ComponentNode<*>, xmlns : string = 'http://www.w3.org/2000/svg') {
+    if (node.type === NODE_TYPE.TEXT) {
+        return;
+    }
+    let ns = xmlns;
+    if (typeof node.props.xmlns === 'string') {
+        ns = node.props.xmlns;
+    } else {
+        node.props.xmlns = ns;
+    }
+
+    if (Array.isArray(node.children) && node.children.length > 0) {
+        node.children.forEach((child : ElementNode | TextNode | ComponentNode<*>) => {
+            addXmlNamespace(child, ns);
```

Comment:
Does this really need to be recursive? Isn't it just the top level svg element which needs the namespace set?

Reply:
The svg element and its children need to be created with the `createElementNS`. 

`document.createElement('path').constructor.name === 'HTMLUnknownElement'`

`document.createElementNS('http://www.w3.org/2000/svg','path').constructor.name === 'SVGPathElement'`